feat(setup): a basic api setup with test data

Mac-Adam opened the pull request.

Mac-Adam committed:
feat(api setup): a basic api setup with simple test db structure and minimal queries

reqill commented on .gitignore:
```diff
@@ -25,4 +25,7 @@ lerna-debug.log*
+package-lock.json
+
+#env
+.env
```
place local .env inside `/pacgkages/drink-buddy-app/config/env.local` for local hosted database

reqill commented on packages/drink-buddy-ui/graphql/context.ts:
```diff
@@ -0,0 +1,17 @@
+import { PrismaClient } from '@prisma/client';
```
Please ad `index.ts` to `/graphql`

Mac-Adam committed:
refactor(moved graphql code and env to proper folders): moved env to config, created graphql/index

env is not yet working so the place where we use env we just use a string now, graphql code moved
from pages folder to graphql folder, left minimal code in pages folder

Mac-Adam committed:
refactor(shortened api/graphql.ts): merged import and export into one line
